JustBrewIt/Pages/Search.cshtml.cs

```diff
@@ -16,44 +16,41 @@ public class SearchModel : PageModel
         public string BreweryType { get; set; }
         [BindProperty]
         public string BreweryCity { get; set; }
-        public bool isSearchCity { get; set; }
+        public bool isSearchCity = false;
 
-        public bool isSearchType { get; set; }
+        public bool isSearchType = false;
```

Comment:
Set boolean variable by default to false, assign it as true eh

Reply:
Hmmmm.  For proper encapsulation, it should have getters and setters.